src/index.html

```diff
@@ -3,8 +3,9 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>Title</title>
-  <script src="https://cdnapisec.kaltura.com/p/1804331/embedPlaykitJs/uiconf_id/51444112"></script>
+  <title>Share Plugin</title>
+<!--  <script src="https://cdnapisec.kaltura.com/p/1804331/embedPlaykitJs/uiconf_id/51444112"></script>-->
+  <script src="http://localhost:8089/kaltura-ovp-player.js"></script>
```

Comment:
the whole point of the demo page is to not need to run scripts from other repos.
also, even locally, player usually wont run on 8089 unless ports 8080-8088 are taken.